contracts/SimpleAuction.sol

```diff
@@ -173,9 +186,8 @@ contract SimpleAuction is /* AccessControl, */ MetaTxContext, ERC1155HolderUpgra
         // Take the new deposit from the new buyer.
         bid.buyer = buyer;
         bid.price = price;
-        _takeDeposit(buyer, price);
```

Comment:
I suppose that is an error to remove this line.

Reply:
seems removed it by mistake